packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -23,6 +23,27 @@ export interface ClusterProps {
    * The VPC where your ECS instances will be running or your ENIs will be deployed
    */
   readonly vpc: ec2.IVpc;
+
+  /**
+   * The service discovery namespace created in this cluster
+   *
+   * @default none
+   */
+  readonly defaultCloudMapNamespace?: NamespaceOptions;
```

Comment:
Shouldn't that have been `CloudMapNamespaceOptions`?

Reply:
Renaming NamespaceOptions was a separate PR that was merged in. Addressed.